src/Compilers/CSharp/Portable/FlowAnalysis/AbstractRegionDataFlowPass.cs

```diff
@@ -4,14 +4,18 @@
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.CSharp.Symbols;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
     // Note: this code has a copy-and-paste sibling in AbstractRegionDataFlowPass.
     // Any fix to one should be applied to the other.
     internal class AbstractRegionDataFlowPass : DataFlowPass
     {
+        // Local functions currently don't implement data-flows-out properly
+        // so we currently fail the analysis if we see a local function.
+        // See https://github.com/dotnet/roslyn/issues/14214
+        private bool _badRegion = false;
```

Comment:
Now that you do analysis at the IDE level, do you need to make any compiler changes at all?